docs/plugins/griffe_doclinks.py

```diff
@@ -76,15 +79,7 @@ def update_docstring(obj: GriffeObject) -> str:
     )
 
 
-def update_docstrings_recursively(obj: GriffeObject) -> None:
-    if obj.docstring:
-        obj.docstring.value = update_docstring(obj)
-    for member in obj.members.values():
-        if not member.is_alias:
-            update_docstrings_recursively(member)
-
-
-class Extension(VisitorExtension):
-    def visit_module(self, node: ast.AST) -> None:
-        module = self.visitor.current.module
-        update_docstrings_recursively(module)
+class UpdateDocstringsExtension(Extension):
+    def on_instance(self, *, node: ast.AST | ObjectNode, obj: GriffeObject) -> None:
```

Comment:
Sorry, upcoming v1 will pass an additional `agent` argument, and parent signatures also declare `**kwargs` so that downstream subclasses declare it too. It means you can remove unused params since they'll be swallowed by kwargs :)

I'll let you check the changelog. I can also send PRs to update everything for upcoming v1! Just tell me ^^

Reply:
Thanks for letting us know. We have our dependencies pinned anyway so it won't break anything until we manually update. I'll make sure to check the changelog and update our extension accordingly!

Au plaisir